src/config/config.rs

```diff
@@ -88,6 +88,43 @@ impl GirVersion {
     }
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum VariantTraitMode {
+    None,
+    Repr,
+    String,
+}
```

Comment:
```suggestion
#[derive(Default, Clone, Copy, Debug, Eq, PartialEq)]
pub enum VariantTraitMode {
    #[default]
    None,
    Repr,
    String,
}
```
instead of a manual default impl?

Reply:
Oh right, I forgot that was a thing now